### PR TITLE
finalizing the view

### PR DESCRIPTION
**Pull request by osbentech**
This is an RoR capstone project named Budget App. It helps users manage day-to-day transactions
below are the features I implemented:

 - A simple page with the name of the App and links to the signup and log-in pages.
 
Sign up and login pages
- The user should be able to register in the app with full name, email, and password
- The user can log into the app using email and password.
- If the user is not logged in, they can't access pages that require the user to be logged in

Home page (categories page)
- When the user logs in, they are presented with the categories page.
- For each category, the user can see their name, icon, and the total amount of all the transactions that belong to that category.
- When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category.
- There is a button "add a new category" at the bottom that brings the user to the page to create a new category.

Transactions page
- For a given category, the list of transactions is presented, ordered by the most recent.
- At the top of the page the user could see the total amount for the category
- There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.
- When the user clicks on the "Back" button (<), the user navigates to the home page.

"Add a new category" page

- The user fills out a form to create a new category, indicating their name and icon (both mandatory).
- The user clicks (or taps) the "Save" button to create the new category, and is taken to the home page on success.
- When the user clicks on the "Back" button (<), the user navigates to the home page.

"Add a new transaction" page

-  The user fills out a form to create a new transaction with:
        name (mandatory)
        amount (mandatory)
        categories (mandatory at least one)
 - The user click (or taps) the "Save" button to create the new transaction, and is taken to the transactions page for that category.
 - When the user clicks on the "Back" button (<), the user navigates to the transactions page for that category.

### CODE REVIEW

File: app/views/devise/registrations/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          <div class="signupname">
+            <%= f.label :name, class: "signupname" %> 
+            <%= f.text_field :name, autofocus: true, autocomplete: "name", class: "signupname" %>
+          </div>
+          <div class="signupemail">
+            <%= f.label :email, class: "block text-sm font-medium text-gray-700" %> <%=
+            f.email_field :email, autofocus: true, autocomplete: "email", class: "semail" %>
+          </div>
+          <div class="signuppass">
+            <%= f.label :password, class: "signup", autocomplete:
```
Comment (hiromataba) [inline app/views/devise/registrations/new.html.erb]: - [ ] Please Let us try to update our signup page to match the original design but styling it just a bit 😉 

- Your design 

image

-Original Design

image

File: app/views/devise/sessions/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class:"emailf" %>
+  </div>
+
+  <div class="pass">
+    <%= f.label :password, class:"sr-only" %>
+
+    <div class="pass">
+      <%= f.password_field :password, autocomplete: "current-password", class:"passf" %>
+    </div>
+  </div>
```
Comment (hiromataba) [inline app/views/devise/sessions/new.html.erb]: - [ ] Please let us update our code and match the original login design 

- Original design 👇 

image

- Your design 

image

File: app/views/devise/sessions/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,44 @@
+<div class="log-container">
+    <div class="mx-auto max-w-lg text-center">
+      <h1 class="login-head">Budget App</h1>
+
+      <p class="lp">
+        Log in to your account to continue
+      </p>
+    </div>
```
Comment (hiromataba) [inline app/views/devise/sessions/new.html.erb]: - [ ] Please update your splash screen to match the original design 👍 

- Your design 👇 

image

- Original design 👇 

image

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @osbentech    :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**